internal/logging/levels.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 The ORC Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+// These levels are defined to categorize what a given log level means.
+// Logs with levels 1-3 are for operators. Log level 4 is for developers.
+const (
+	// Status logs are always shown. Any customer facing message should
+	// fall into this level. Startup and shutdown messages should fall into
+	// this log level.
+	Status = 1
+	// Info logs should log events that happen only once per reconcile loop.
+	// Example: "Reconciling OpenStack resource delete"
+	Info = 2
+	// Verbose logs might be of interest to a system administrator trying to get a
+	// more detailed understanding of why events are happening.
+	// Example: "web-download is not supported because..."
```

Comment:
```suggestion
	// Verbose logs provide additional context for an administrator trying to understand why an action
	// be occurring or not occuring. It should produce logs on every reconcile attempt.
	// Example: "web-download is not supported because..."
```